src/test/java/unit/Workspace.java

```diff
@@ -162,6 +162,8 @@ void statusDescribeListReflectUpdate() throws IOException {
         status.workspace.description,
         "status matches updated workspace description");
 
+    TestCommand.runCommandExpectSuccess("workspace", "delete-property", "--keys=key");
```

Comment:
The way unit tests work:

- Do A
- Check A
- Do B
- Check B
- Do C
- Check C

Here, you're calling `terra workspace delete` in the middle of "Check B".

Also, each of these tests is testing something specific. The name of this test is `statusDescribeReflectsSet`. Based on this name, this test:

- `terra workspace set --id=123`
- `terra workspace status` returns status for workspace 123
- `terra workspace describe` returns workspace 123

Let's add a new test for this new property command.

```
  @Test
  @DisplayName("delete property")
  void deleteProperty() throws IOException {
   Create a workspace with 3 properties
   Call `terra workspace delete-property` for 2 properties
   Check workspace only has 1 property
```

Reply:
Thanks Melissa. I was also hesitate when I write this. Yes. your idea is much more clear. I just correct both tickets with the new unit test.